sendmail.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Thoth's integration tests
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/
+
+"""Send e-mail report, used to send reports to our mailing list."""
+
+from datetime import date
+from email.mime.text import MIMEText
+import os
+import smtplib
+
+_BEHAVE_REPORT_FILE = "behave-report.html"
+_DEPLOYMENT_NAME = os.getenv("THOTH_DEPLOYMENT_NAME", "N/A")
+_EMAIL_SMTP_SERVER = os.getenv("THOTH_EMAIL_SMTP_SERVER", "smtp.corp.redhat.com")
+_EMAIL_TO = os.getenv("THOTH_EMAIL_TO", "aicoe-thoth@redhat.com")
```

Comment:
shall we send it to aicoe-thoth-devops?

Reply:
Yes, that will be probably better not to produce too much spam. I'll open a separate PR for this as this is already part of https://github.com/thoth-station/integration-tests/pull/109